docs/reference/edot-sdks/android/configuration.md

```diff
@@ -0,0 +1,380 @@
+
+```kotlin
+class MyApp : android.app.Application {
+
+   override fun onCreate() {
+      super.onCreate()
+      val agent = ElasticApmAgent.builder(this)
+         // ...
+         .setProcessorFactory(factory)
+         .build()
+   }
+}
+```
+
+The factory is called once during initialization and needs to provide a processor per signal. Each processor-provider method within the factory contains the configured exporter for that signal as an argument so that it's included into the processor as its delegate exporter.
+
+### Internal logging policy
+
+:::{note}
+Not to be confused with OpenTelemetry's [log signals](https://opentelemetry.io/docs/concepts/signals/logs/). The internal logging policy is about the agent's internal logs that you should see in [logcat](https://developer.android.com/studio/debug/logcat) only.
+:::
+
+The agent creates logs using [Android's Log](https://developer.android.com/reference/android/util/Log) type to notify about its internal events, so that you can check them out in [logcat](https://developer.android.com/studio/debug/logcat) for debugging purposes. By default, all logs are printed for a debuggable app build. However, in the case of non-debuggable builds, only logs at the `INFO` level and higher are printed.
+
+If you want to show specific logs from the agent, or even turn off logs altogether, you can do so by providing your own `LoggingPolicy` configuration. The following example shows how to allow all logs of level `WARN` and higher to be printed, whereas those lower than `WARN` are ignored.
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            // ...
+            .setInternalLoggingPolicy(LoggingPolicy.enabled(LogLevel.WARN))
+            .build()
+    }
+}
+```
+
+## Dynamic configuration
+
+Dynamic configuration settings are available from an already built [agent](https://github.com/elastic/apm-agent-android/blob/main/agent-sdk/src/main/java/co/elastic/otel/android/ElasticApmAgent.kt).
+
+### Update export connectivity
+
+You can change any of the configuration values provided as part of the [export connectivity](#export-connectivity) setters, at any time by setting a new [ExportEndpointConfiguration](https://github.com/elastic/apm-agent-android/blob/main/agent-sdk/src/main/java/co/elastic/otel/android/connectivity/ExportEndpointConfiguration.kt) object, which overrides them all. For example:
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            // ...
+            .build()
+        agent.setExportEndpointConfiguration(configuration)
+    }
+}
+```
+
+## Provide config values from outside of your code
```

Comment:
I have a similar comment on this complementary information as I had on the getting-started page, where we're mixing these "mini-tutorials", which provide tangential information, with topics that I think should be focused on what the users should expect to see on this page.

It might seem like the topics from this part of the file until its end are configurable parameters of the agent, but that's not the case. It's mostly tips on ways users can organize their projects to be able to provide config parameters from different places (such as environment vars and properties files). This information is less about the configurable parameters from the agent and more about addressing common use-cases and questions from our users.